ArgoTests/JSON/JSONFileReader.swift

```diff
@@ -1,15 +1,14 @@
 import Foundation
+import Runes
 
 class JSONFileReader {
   class func JSON(fromFile file: String) -> AnyObject? {
-    let path = NSBundle(forClass: self).pathForResource(file, ofType: "json")
+    return data(fromFile: file) >>- { NSJSONSerialization.JSONObjectWithData($0, options: NSJSONReadingOptions(0), error: nil) }
```

Comment:
My comment above about the reading options applies here as well. I think this API could be improved in two ways.

1. Change this function to also take a `JSONDeserializer` to use for the deserialization step.
2. Move this function to `JSONDeserializer`